opentracing-spring-web-handler-interceptor/src/main/java/io/opentracing/contrib/spring/web/interceptor/InterceptorSpanWrapper.java

```diff
@@ -0,0 +1,26 @@
+package io.opentracing.contrib.spring.web.interceptor;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.web.servlet.filter.SpanWrapper;
+
+/**
+ * @author Pavol Loffay
+ */
+class InterceptorSpanWrapper extends SpanWrapper {
+
+    private Object handler;
```

Comment:
What is this handler? Can't you add a parameter to this class, so that you have a real type, instead of using `Object`?